functions/openhabian.bash

```diff
@@ -9,6 +9,20 @@ get_git_revision() {
   echo "[$branch]$latesttag-$revcount($shorthash)"
 }
 
+apt_update() {
+  echo -n "$(timestamp) [openHABian] Updating Linux package information ... "
+  apt-get -q update >/dev/null 2>&1 &
+  PID_APT=$!
+}
+
+wait_for_apt_to_finish_update() {
+  if [ -v PID_APT ]; then
+    wait -f ${PID_APT} 2>/dev/null
+  else
+    apt_update
```

Comment:
...and wait for completion
so the better approach would be
`if not ... then apt_update; fi
wait -f ...`

Reply:
right